libraries/icms/core/DataFilter.php

```diff
@@ -980,7 +980,7 @@ static private function priv_checkVar($data, $type, $options1, $options2) {
 					break;
 
 					case "host":
-						$valid = filter_var($data, FILTER_VALIDATE_URL, FILTER_FLAG_HOST_REQUIRED);
+						$valid = filter_var($data, FILTER_VALIDATE_URL);
```

Comment:
I'm curious about the situation here. It makes sense that if the validation operation is looking for 'host' that we would want the host to be required. But I assume you have an unconventional URL? Can you give an example of where this code is/was problematic?

Reply:
https://php.watch/versions/8.0/filter_var-flags 
The FILTER_FLAG_HOST_REQUIRED flag got removed in PHP8, so it throws fatal error if we keep it.

Reply:
Aha, I wondered about that. Did not see a reference in the documentation. But then, there it is, in the notes at the bottom of the page. Cool.